<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MockTradeForm</class>
 <widget class="QWidget" name="MockTradeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模拟交易</string>
  </property>
  <widget class="QPushButton" name="pbtnBuy">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>买入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnSell">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>卖出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnCondition">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>条件单</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dbspbFeeOpen">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>100.000000000000000</double>
   </property>
   <property name="value">
    <double>100.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="lab_assets">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>初始资金:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>可用资金:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开仓手续费:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dbspbFeeOpen_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>500.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>100.000000000000000</double>
   </property>
   <property name="value">
    <double>300.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dbspbBegCapital">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <double>12000.000000000000000</double>
   </property>
   <property name="value">
    <double>12000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_cur_capital">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>权益:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>平仓手续费:</string>
   </property>
  </widget>
  <widget class="QTableView" name="table_view_record">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>561</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lab_status">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>571</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>status</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnclose">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>平仓</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_long_pos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
